presto-tests/src/main/java/com/facebook/presto/tests/DistributedQueryRunner.java

```diff
@@ -319,6 +319,11 @@ public final void close()
 
     private void cancelAllQueries()
     {
+        // this can happen when an exception is thrown in the constructor
```

Comment:
This isn't needed anymore since I fixed it not to call `close()` in the constructor